### PR TITLE
Fix most undefined references

### PR DESCRIPTION
**Pull request by arichardson**
Significantly reduces the number of warnings produced by the build - we are down to the following after this PR:

```
LaTeX Warning: Text page 31 contains only floats.
LaTeX Warning: `!h' float specifier changed to `!ht'.
LaTeX Warning: `!h' float specifier changed to `!ht'.
LaTeX Warning: Hyper reference `sailRISCVzMAX' on page 178 undefined on input l
LaTeX Warning: Hyper reference `sailRISCVzEzyFetchzyAddrzyAlign' on page 207 un
LaTeX Warning: Hyper reference `sailRISCVzEzyFetchzyAddrzyAlign' on page 209 un
LaTeX Warning: Hyper reference `sailRISCVzEzyFetchzyAddrzyAlign' on page 210 un
LaTeX Warning: Hyper reference `sailRISCVzcapszyperzycachezyline' on page 214 u
LaTeX Warning: Hyper reference `sailRISCVzEzyLoadzyAddrzyAlign' on page 214 und
LaTeX Warning: Hyper reference `sailRISCVztranslateAddr' on page 214 undefined 
LaTeX Warning: Hyper reference `sailRISCVzRead' on page 214 undefined on input 
LaTeX Warning: Hyper reference `sailRISCVzTRzyFailure' on page 214 undefined on
LaTeX Warning: Hyper reference `sailRISCVzEzyExtension' on page 214 undefined o
LaTeX Warning: Hyper reference `sailRISCVzinternalzyerror' on page 214 undefine
LaTeX Warning: Hyper reference `sailRISCVzTRzyFailure' on page 215 undefined on
LaTeX Warning: Hyper reference `sailRISCVzTRzyAddress' on page 215 undefined on
LaTeX Warning: Hyper reference `sailRISCVzEzyExtension' on page 215 undefined o
LaTeX Warning: Hyper reference `sailRISCVzMemoryOpResult' on page 215 undefined
LaTeX Warning: Hyper reference `sailRISCVzMemValue' on page 215 undefined on in
LaTeX Warning: Hyper reference `sailRISCVzMemException' on page 215 undefined o
LaTeX Warning: Hyper reference `sailRISCVzMemValue' on page 215 undefined on in
LaTeX Warning: Hyper reference `sailRISCVzmemzyreadzycap' on page 215 undefined
LaTeX Warning: Hyper reference `sailRISCVzMemException' on page 215 undefined o
LaTeX Warning: Hyper reference `sailRISCVzMemException' on page 215 undefined o
LaTeX Warning: Hyper reference `sailRISCVzMemValue' on page 215 undefined on in
LaTeX Warning: Hyper reference `sailRISCVzMemValue' on page 215 undefined on in
LaTeX Warning: Hyper reference `sailRISCVzMemException' on page 215 undefined o
LaTeX Warning: Hyper reference `sailRISCVzMemValue' on page 215 undefined on in
LaTeX Warning: Hyper reference `sailRISCVzcapszyperzycachezyline' on page 215 u
LaTeX Warning: Hyper reference `sailRISCVzexecute' on page 241 undefined on inp
LaTeX Warning: Hyper reference `sailRISCVzexecute' on page 242 undefined on inp
LaTeX Warning: Hyper reference `sailRISCVzRISCVzyJALR' on page 242 undefined on
LaTeX Font Warning: Font shape `T1/ptm/m/scit' undefined
LaTeX Font Warning: Font shape `T1/ptm/m/scit' undefined
LaTeX Font Warning: Font shape `T1/ptm/m/scit' undefined
LaTeX Font Warning: Font shape `T1/ptm/m/scit' undefined
LaTeX Font Warning: Font shape `T1/ptm/m/scit' undefined
LaTeX Warning: `h' float specifier changed to `ht'.
LaTeX Warning: There were undefined references.
```

### CODE REVIEW

File: chap-architecture.tex [hunk 1]
```diff
@@ -1616,12 +1616,11 @@ \section{Capability-Aware Instructions}
...
 These instructions modify capability-register fields, setting them to values
 moved from integer registers, subject to constraints such as monotonicity and
-representability: \insnref{CAndAddr}, \insnref{CAndPerm},
-\insnref{CClearTag},
+representability: \insnref{CAndPerm}, \insnref{CClearTag},
 \insnref{CIncOffset}, \insnref{CIncOffsetImm},
 \insnref{CSetAddr}, \insnref{CSetBounds},
 \insnref{CSetBoundsExact}, \insnref{CSetBoundsImm},
-\insnref{CSetFlags}, and
+\insnref{CSetFlags}, \insnref{CSetHigh}, and
```
Comment (bsdjhb) [inline chap-architecture.tex:1623]: Add CSetHigh here appears to be an unrelated change?

**Review by bsdjhb — APPROVED** [review]
LGTM modulo the nit about CSetHigh.

File: chap-architecture.tex (hunk 1 above)
Comment (arichardson) [reply chap-architecture.tex:1623]: Ah yeah, I replaced the no longer valid instruction with the newly added modification one. Will update to just remove the CAndAddr.
